README.md

```diff
@@ -25,29 +25,40 @@ A full reference for this library is available [here](./reference.md).
 from cartesia import Cartesia
 import os
 
-client = Cartesia(api_key=os.environ.get("CARTESIA_API_KEY"))
+client = Cartesia(api_key=os.getenv("CARTESIA_API_KEY"))
 
 # Get all available voices
 voices = client.voices.list()
 print(voices)
 
 # Get a specific voice
 voice = client.voices.get(id="a0e99841-438c-4a64-b679-ae501e7d6091")
-print("The embedding for", voice["name"], "is", voice["embedding"])
+print("The embedding for", voice.name, "is", voice.embedding)
 
-# Clone a voice using filepath
-cloned_voice_embedding = client.voices.clone(filepath="path/to/voice")
+# Clone a voice using file data
+cloned_voice = client.voices.clone(
+    clip=open("path/to/voice.wav", "rb"),
+    name="Test cloned voice", 
+    language="en",
+    mode="similarity",  # or "stability"
+    enhance=False, # use enhance=True to clean and denoise the cloning audio
+    description="Test voice description"
+)
 
 # Mix voices together
-mixed_voice_embedding = client.voices.mix(
-    [{ "id": "voice_id_1", "weight": 0.5 }, { "id": "voice_id_2", "weight": 0.25 }, { "id": "voice_id_3", "weight": 0.25 }]
+mixed_voice = client.voices.mix(
+    voices=[
+        {"id": "voice_id_1", "weight": 0.25},
+        {"id": "voice_id_2", "weight": 0.75}
+    ]
 )
 
-# Create a new voice
+# Create a new voice from embedding
```

Comment:
we can probably just kill this section?

Reply:
good to keep it in for backcompat? i don't have strong opinions

Reply:
Well it's no longer clear how to get the value for `embedding=[...]`, which is why I think we should remove it

Reply:
```
voice = client.voices.get(id="a0e99841-438c-4a64-b679-ae501e7d6091")
print("The embedding for", voice.name, "is", voice.embedding)
```

Reply:
oh I guess `client.voices.mix` still returns an embedding

Reply:
but i agree, there's no use case for it anymore since we've fixed the cloning API lol

Reply:
ah, i spoke too soon